assets/styles/Labels.less

```diff
@@ -36,8 +36,7 @@
     font-weight: normal;
     color: rgb(255 255 255 / 65%);
     font-family: monospace;
-    margin: auto;
-    margin-left: 3px;
+    margin: auto auto auto 3px;
```

Comment:
This is in fact shorter, but less readable. I don't usually remember which of the four arguments is what (less when there are three). Right now I see it as "everything with auto but left with 3px", so much easier. This kind of changes are handled by the CSS postprocessor so they have no impact.

There are a few cases of this

Reply:
Fair enough, I didn't know LESS did optimization

Reply:
There's quite a few places where you were already using this syntax; for consistency, should I expand those too?

Reply:
> There's quite a few places where you were already using this syntax; for consistency, should I expand those too?

As you wish

Reply:
Well, it's your project, so it should be as you wish, but so be it 😅